interfaces/Portalicious/src/app/utils/chart.ts

```diff
@@ -4,6 +4,9 @@ import ChartDataLabels from 'chartjs-plugin-datalabels';
 export function registerChartDefaults() {
   const documentStyle = getComputedStyle(document.documentElement);
 
+  Chart.defaults.font.family = 'Montserrat, sans-serif';
```

Comment:
There is no magical `var(--font-display)` Tailwind-utility, right?

Or like with the `--text-color` below?

Using the TailwindConfig as-is wouldn't really work unfortunately... :/

Reply:
No, I tried to pull it out but couldn't find a way to make it happen.

Worst case scenario the fonts in the chart are slightly off - feels like a fair risk to me given that we literally have one chart on the portal (for now).